README.md

```diff
@@ -50,7 +51,8 @@ class Customer(models.Model):
         app_label = 'myapp'
 
 class PreferredCustomer(pg.View):
-    projection = ['myapp.Customer.*']
+    projection = ['myapp.Customer.*',]
+    dependencies = ['myapp.Customer',]
```

Comment:
However, we use a custom `__metaclass__` here that could `pop` the additional attributes before the parent `__new__` is called, so maybe that could work?